webapp/boards/package.json

```diff
@@ -63,17 +63,6 @@
 		"trim-newlines": "^4.0.2"
 	},
 	"devDependencies": {
-		"@babel/cli": "7.17.6",
-		"@babel/core": "7.17.8",
-		"@babel/plugin-proposal-class-properties": "7.16.7",
-		"@babel/plugin-proposal-object-rest-spread": "7.17.3",
-		"@babel/plugin-proposal-optional-chaining": "7.16.7",
-		"@babel/plugin-syntax-dynamic-import": "7.8.3",
-		"@babel/polyfill": "7.10.4",
-		"@babel/preset-env": "7.16.11",
-		"@babel/preset-react": "7.16.7",
-		"@babel/preset-typescript": "7.16.7",
-		"@babel/runtime": "7.17.8",
```

Comment:
🤩

Reply:
That's a good point. I was trying to avoid making any Boards-specific changes, but maybe I should avoid making any changes to this folder.

I'll see if anyone has any thoughts on that